### ISSUE TITLE
Download img to active path

### ISSUE THREAD

**Issue by niXta1** [teracow/googliser#21]
Hi!

Thanks for this script! I'm running macOS 14.10.0.

How do I download the image to the active folder path?
I played around with -o "./" etc. and had a look in the script, without modifying it, I don't know how to do it.

**Comment by teracow** [comment]
Hello, I've just tried `-o ./` and it works fine here. 

Using BASH v4 to test:
```
$ bash --version
GNU bash, version 4.4.12(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
```
Can you please post your debug log?

**Comment by niXta1** [comment]
Uppdated info about OS.
This is the error in the log:
`! create ouput directory [./]: 'failed! Directory already exists!'`
I get the same error if I set it to -o /complete-dir

Maybe a check before creating the folder will solve it?

**Comment by teracow** [comment]
If you're able, can you please post the debug log?

**Comment by niXta1** [comment]
> started: 'Fre  4 Jan 2019 16:14:49 CET'
? $script_details: 'googliser.sh - 2018-10-17 PID:[83694]'
? $user_parameters_raw: '-p Dean Koontz - Vaktare ljudbok --debug -f 15 -P 1 -N -n 1 -r 10 -t 15 -o /Volumes/media/audio/Ljudbocker/Dean_Koontz-Vaktare-AUDIOBOOK-SE-2015-iND'
\ [ValidateParameters]: 'entry'
/ [ValidateParameters]: 'exit'
= environment: '*** parameters after validation and adjustment ***'
? $user_query: 'Dean Koontz - Vaktare ljudbok'
? $images_required: '1'
? $fail_limit: '40'
? $parallel_limit: '1'
? $timeout: '15'
? $retries: '10'
? $upper_size_limit: '0'
? $lower_size_limit: '1000'
? $create_gallery: 'false'
? $gallery_title: ''
? $condensed_gallery: 'false'
? $save_links: 'false'
? $colour: 'false'
? $verbose: 'true'
? $debug: 'true'
? $skip_no_size: 'false'
? $remove_after: 'false'
? $lightning: 'false'
? $min_pixels: ''
? $aspect_ratio: ''
? $image_type: ''
? $usage_rights: ''
? $input_pathfile: ''
? $output_path: '/Volumes/media/audio/Ljudbocker/Dean_Koontz-Vaktare-AUDIOBOOK-SE-2015-iND'
? $links_only: 'false'
? $dimensions: ''
= environment: '*** internal parameters ***'
? $google_max: '1000'
? $temp_path: '/tmp/googliser.83694.jKj'
$ required program is available: 'wget'
! optional program is unavailable: 'identify'
? $safe_query: 'Dean_Koontz_-_Vaktare_ljudbok'
? $safe_path: '/Volumes/media/audio/Ljudbocker/Dean_Koontz-Vaktare-AUDIOBOOK-SE-2015-iND'
? $target_path: '/Volumes/media/audio/Ljudbocker/Dean_Koontz-Vaktare-AUDIOBOOK-SE-2015-iND'
~ $gallery_title was unspecified so set as: 'Dean Koontz - Vaktare ljudbok'
! create ouput directory [/Volumes/media/audio/Ljudbocker/Dean_Koontz-Vaktare-AUDIOBOOK-SE-2015-iND]: 'failed! Directory already exists!'
T [googliser.sh] elapsed time: '00h:00m:01s'
< finished: 'Fre  4 Jan 2019 16:14:50 CET'

**Comment by teracow** [comment]
I've just cranked-up my macOS VM and I'm seeing the same problem when using `-o ./` :+1: 

I'll muck-about with it and see what I can do. :)

**Comment by teracow** [comment]
I've worked out why my initial testing passed. 

The directory test works by looking for the target path, and checking for more than zero files in it. If there are no files, the script will use that existing directory. If there are 1 or more files in that existing directory, an error is thrown. 

In my usual semi-organised fashion, I tested functionality by creating a new directory, changing into it and running googliser with `-o ./`. Of course, this ran fine as there was nothing in that directory (and I run googliser via $PATH). :D

But that's not the same as running googliser when [googliser.sh] is in that same directory.
